src/GdbServer.cc

```diff
@@ -53,20 +54,6 @@ static const int DBG_COMMAND_MSG_DELETE_CHECKPOINT = 0x02000000;
 
 static const uint32_t DBG_COMMAND_PARAMETER_MASK = 0x00FFFFFF;
```

Comment:
Yes, I plan on submitting that soon. I just wanted to make sure we were ok with this before I rebased my changes.